### PR TITLE
Moved Integration Tests and Created Controller Unit Tests

### PR DESCRIPTION
**Pull request by AndyZheng430**
### **Changes**
[+] Added Folders for Unit Test and Integration Test and moved tests into respective folders
[+] Added Unit test to controllers
[+] pom.xml reports only unit tests

### CODE REVIEW

File: budget-service/src/test/java/com/skillstorm/budgetservice/unit/models/BudgetTest.java [hunk 1]
```diff
@@ -78,11 +81,11 @@ void testCreatedTimeStamp() {
         assertEquals(now, budget.getCreatedTimeStamp());
     }
 
-    @Test
-    void testPrePersist() {
-        budget.onCreate();
-        assertNotNull(budget.getCreatedTimeStamp());
-    }
+    // @Test
```
Comment (AndyZheng430) [inline budget-service/src/test/java/com/skillstorm/budgetservice/unit/models/BudgetTest.java:84]: Can't test private or protected model class methods

**Comment by AndyZheng430** [comment]
Is there a standard on how to separate unit tests and integration tests? 
I separated them by creating 2 folders and putting tests in there; doing so does not allow me to tests private/protected methods since they are in different packages. 
Example: 
model class would be in the package com.skillstorm.budgetservice.models 
unit tests for the models are in package com.skillstorm.budgetservice.unit.models
unit tests can't test private/protected methods of model class

**Review by brittshook — APPROVED** [review]
@AndyZheng430 Looks good, maybe just rename the files to say either Unit or Integration testing for clarity in your next PR

**Review by bbvicente — APPROVED** [review]
Looks good!

**Pull request merged**